spec/models/taxon_spec.rb

```diff
@@ -115,6 +115,16 @@
   it "should strip leading space" do
     expect( Taxon.make!( name: "   Leading space" ).name ).to eq "Leading space"
   end
+  
+  it "should prevent creating a taxon with a rank coarser than the parent" do
+    parent = Taxon.make!( rank: Taxon::GENUS)
```

Comment:
Missing a space. Also, when writing new Ruby code, please use the new-style hash syntax and double quotes whenever possible. So below should be `taxon = Taxon.new( name: "balderdash", rank: Taxon::FAMILY, parent: parent )`